providers/providers.go

```diff
@@ -58,6 +58,26 @@ func init() {
 	}
 
 	LastProviderInstall = time.Now().Unix()
+
+	// Initialize the global coordinator instance
+	coordinator := newCoordinator()
+	Coordinator = coordinator
+
+	// Load the schema for all the active providers
+	providers, err := ListActive()
+	if err != nil {
+		log.Error().Err(err).Msg("failed to list active providers")
+		return
+	}
+
+	for name := range providers {
+		schema, err := Coordinator.LoadSchema(name)
```

Comment:
This code will read all schemas that are installed on the system by default on startup. For regular use of scan/run/shell I had originally avoided this, so that we only load schemas we need and dynamically load additional schemas (making startup more lightweight). 

Is it necessary to load all schemas for this change?
If that is the case and we require them here, would you mind adding a comment so that we change this back to dynamic loading later on (I'm sure I'll forget otherwise :see_no_evil: ) (something like `FIXME: dynamically load providers on startup`).